Add a Dockerfile image to run the different tutorials under baleine.

haysberg opened the pull request:
I added a Dockerfile with all the necessary binaries to run the tutorials.

Stuff like ping, ip or netcat.

parmentelat commented:
9 commits pour ajouter un fichier ça fait vraiment beaucoup
ça parait une occasion unique d'apprendre à faire un rebase ;)
ta mission consiste donc à récrire tout ça en un commit, et `rebase -i` est ton ami

Commit:
Created Docker image for r2lab


Changed main to master


Fix Dockerfile


Image now has a symlink pointing to the shell utils


Appending prompt instead of overwriting bashrc


Faster build + Timezone setup


Moved baleine Dockerfile to correct position


Deleting GH Action

haysberg commented:
Est-ce que c'est mieux comme ça ?

parmentelat commented:
bin non, avant il y avait 9 commits à merger et maintenant il y en a 11...
en fait celui qui m'intéresse c'est juste c653e93b58bce9feea08165252a0785c7d11152c
c'était très bien mais il fallait pas remerger les 9 autres évidemment...

---
tu vas faire ça dans ton repo
(tu pars bien d'un repo clean pour rien perdre dans la bataille
 c-a-d: fais un stash avant si nécessaire)

```
git reset --hard c653e93b
git push --force
```

et après je pourrai merger

(bon sinon je peux aussi me débrouiller tout seul sans passer par github mais autant que tu apprennes à t'en servir...)

haysberg commented:
Parfait ! Ça marche mieux comme ça.
Je pensais que je pouvais seulement squash les commits, mais je crois que j'aurais dû faire ça avant de créer la pull request.

Maintenant on est bons je crois.
